cmd/ship-it-api/main.go

```diff
@@ -43,13 +44,14 @@ func main() {
 	dd := dogstatsd.New("wattpad.ship-it.", logger)
 	go dd.SendLoop(time.Tick(time.Second), "udp", cfg.DataDogAddress())
 
-	k8s, err := k8s.New(ctx, cfg.HelmReleasesResync())
+	k8s, err := k8s.New(ctx, cfg.HelmReleasesResync(), logger)
 	if err != nil {
 		logger.Log("error", err)
 		os.Exit(1)
 	}
 
-	svc := service.New(k8s)
+	gitClient := github.New(ctx, cfg.GithubOrg, cfg.GithubToken)
```

Comment:
git and github are different things. Even though it's very obvious from context that this client is for github, I'd suggest naming this `githubClient`.